Library/Homebrew/cmd/test-bot.rb

```diff
@@ -639,10 +639,9 @@ def homebrew
         tests_args << "--coverage" if ruby_two && ENV["TRAVIS"]
         test "brew", "tests", *tests_args
         test "brew", "tests", "--no-compat"
+        test "brew", "readall", "--syntax" if ruby_two
```

Comment:
Do you know why this is limited to Ruby 2.0+? I just ran it both with the system Ruby (on 10.11) and with a brewed Ruby 1.8.7 and it completed without any issues, thus I would suggest to make this unconditional.